components/table/interface.tsx

```diff
@@ -219,6 +219,8 @@ export interface TableProps<T> extends WithStore {
   getPopupContainer?: (triggerNode: HTMLElement) => HTMLElement;
 }
 
+export type TableProps<T> = WithStore & TablePropsWithoutStore<T>;
```

Comment:
It will breaks `interface A<T> extends TableProps<T> {}`

`WithStore` should just be `Partial`